request_listener/curl.go

```diff
@@ -0,0 +1,67 @@
+package request_listener
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+func CurlLogDumper(request *http.Request) {
+	if request == nil {
+		return
+	}
+	cmd, err := GetCurlCommand(request)
+	if err != nil {
+		log.Println("cannot print curl command", err)
+		return
+	}
+	log.Println(cmd)
+}
+
+func GetCurlCommand(req *http.Request) (*Cmd, error) {
+	cmd := &Cmd{
+		"curl",
+		"-X", escape(req.Method),
+		escape(req.URL.String()),
+	}
+
+	if req.Body != nil {
+		body, err := ioutil.ReadAll(req.Body)
```

Comment:
This is kind of dangerous when the user sends large files don't you think ?

Maybe add a warning in the doc for now ?

Reply:
verbose/heavy is more appropriate than `dangerous`.
And all miss used code/library can lead to perf or data lost
This type of data Dumper should be used with some filter.

Yeah i gonna add doc, have you any words to give?